lib/stack-layers.js

```diff
@@ -86,10 +88,11 @@ module.exports = function(element, id, side, layers, drills, outline, useOutline
     var cfMaskId = idPrefix + 'cf-mask'
     var cfMaskAttr = {id: cfMaskId, fill: '#fff', stroke: '#fff'}
     var cfMaskShape = smLayerId
-      ? [useLayer(element, smLayerId)]
+      ? [element('use', {'xlink:href': '#' + smLayerId, fill: '#fff', stroke: '#fff'})]
```

Comment:
They are definitely necessary. Seems to work just on the group as well though though https://github.com/kasbah/pcb-stackup-inkscape-compat/commit/0eb79042c8f77ec14729d266e69a69b6af351d8c. Should I change it to that?